lms/startup.py

```diff
@@ -105,43 +107,10 @@ def enable_stanford_theme():
 
 def enable_microsites():
```

Comment:
@chrisndodge I vaguely remember something about the jenkins test runner needing it.